### DISCUSSION TITLE
College-level course materials

### DISCUSSION THREAD

**Discussion by MikeMcPherson** [alanbjohnston/CubeSatSim#237] [Q&A]
Hi everyone

I volunteer with the Spacecraft Design program at the University of Virginia.  We are working on incorporating the CubeSatSim into the capstone two-semester course.  Is there anyone else out there working on college-level instructional materials using the CubeSatSim?

Thanks.  Mike KQ9P

**Comment by alanbjohnston** [comment]
Hi Mike,

I have used the CubeSatSim multiple times in our Electrical Engineering Senior Design Capstone, which goes over 3 semesters.  I have the team of students redesign an aspect of the current design, such as the boost converter/regulator or battery charging circuit as their project.

I know it has been used in other universities as well.  One use in senior design was at the University of Tennessee Knoxville.

You can email me directly for some details ku2y@arrl.net

I have also used it in introductory freshman Electrical and Computer Engineering classes where the students just build and operate the CubeSatSims.
